### DISCUSSION TITLE
【教程】如何提升绳网等级

### DISCUSSION THREAD

**Discussion by user1-cloud** [share121/inter-knot#1467] [其他]
![封面](https://github.com/user-attachments/assets/34aa95c0-5071-403c-90c2-b27ff6cad30b)
## 终于可以提升绳网等级了！
众所周知，绳网等级现在与GitHub账号上的repository数量绑定了，所以在这里教各位新来的绳匠如何创建repository[^1]（PC端）

***

1. 在GitHub页面的右上角可以看到一个加号，单击它
![image](https://github.com/user-attachments/assets/90badfdf-6de2-4bfb-95b9-dd044990880c)

***

2. 单击New repository
![image](https://github.com/user-attachments/assets/4d10c9f4-fa8f-4cb3-8b7f-fca212dff493)

***

3. 在“Repository name”一栏中随便输入点东西作为所创建的repository的名字
![image](https://github.com/user-attachments/assets/edbbdd52-6e3f-4feb-ac01-0a691211172c)

***

4. 划到最底部并单击“Create repository”
![image](https://github.com/user-attachments/assets/450a044e-de74-4b64-a29f-560cbd61fd71)

***

5. 回到绳网，你会惊讶地发现你居然升了一级！

***

> [!TIP]
> 一位大佬给出了一个更高级的方法#1489

[^1]:中文：仓库

**Comment by capacitor1** [comment]
GitHub：怎么突然多了一堆空白项目

doge

**Comment by ZMJ2333** [comment]
### 顶！

**Comment by bugmaibear** [comment]
我补充一点这个东西下面的选项必须要选“公用”而不能选“个人”

**Comment by canifish** [comment]
顶
